servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -139,7 +167,14 @@ RD::DataFormat RenderForwardMobile::_render_buffers_get_color_format() {
 	// Using 32bit buffers enables AFBC on mobile devices which should have a definate performance improvement (MALI G710 and newer support this on 64bit RTs)
 	// NO ALPHA and unsigned float.
 	// @TODO No alpha is an issue, recommendation here is to add a second RT for alpha
-	return RD::DATA_FORMAT_B10G11R11_UFLOAT_PACK32;
+	return RD::DATA_FORMAT_A2B10G10R10_UNORM_PACK32;
+}
+
+bool RenderForwardMobile::_render_buffers_can_be_storage() {
+	// Using 32bit buffers enables AFBC on mobile devices which should have a definate performance improvement (MALI G710 and newer support this on 64bit RTs)
+	// NO ALPHA and unsigned float.
```

Comment:
The remark about alpha can be removed here as well, no longer applies

Reply:
I am not sure, even if this format has alpha, 4 bits may not be enough for a lot of people so probably better to leave it for the time being, as we may need a different solution.